examples/app-python/README.md

```diff
@@ -0,0 +1,39 @@
+# Example Python Payload Application
+
+This directory contains an example Payload Application using the Python.
+This is not production-ready code, but it is useful for learning how the platform functions.
+
+## Quickstart
+
+In the Antaris Cloud Platform, create a TrueTwin Satellite with a remote payload and download the associated config.
+Place that downloaded zip file in this directory.
+
+Build the app using the following command:
+
+```
+docker build --platform=linux/amd64 -t satos-payload-example-app-python .
+```
+
+Next, we can run the application in a container. The command below assumes that `CONFIG` is set to the name of the downloaded file in your current working directory:
+
+```
+docker run --platform=linux/amd64 -e CONFIG=$CONFIG -v $(pwd):/workspace -it satos-payload-example-app-python 
```

Comment:
After copying zip file in current dir and providing proper paths, I got following error =

**$ export CONFIG=_Packages_upload_RahulPayload-1684750606-aaa28e2d01

$ docker run --platform=linux/amd64 -e CONFIG=$CONFIG -v $(pwd):/home/rahul/projects/payload_git/test/SatOS-Payload-SDK -it satos-payload-example-app-python

unzip:  cannot find or open /workspace/_Packages_upload_RahulPayload-1684750606-aaa28e2d01, /workspace/_Packages_upload_RahulPayload-1684750606-aaa28e2d01.zip or /workspace/_Packages_upload_RahulPayload-1684750606-aaa28e2d01.ZIP.**